### PR TITLE
New package: hitch-1.4.1

### PR DESCRIPTION
**Pull request by jannisch**
Please tell me in which of the 3 ways the example config file should be installed:

1. (currently used): default path `/usr/share/doc/hitch/hitch.conf.example`
2. with `vsconf`
3. with `vconf` directly into /etc

The user needs to edit the config anyway because of the custom TLS certificate.

### CODE REVIEW

**Comment by jannisch** [comment]
1.4.1 was released in the meantime, please don't merge yet.

**Comment by jannisch** [comment]
Updated, I hope that I did not make a mistake when changing the commit message.

**Comment by Vaelatern** [comment]
I'd be happy with a `vconf` is the config can be used right out of the box.
Otherwise the default is just fine.

**Comment by jannisch** [comment]
Thanks, so no change is required I guess (does not work out of the box).

File: srcpkgs/hitch/template [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+build_style=gnu-configure
+hostmakedepends="pkg-config"
+makedepends="libev-devel libressl-devel"
+short_desc="Libev-based high performance SSL/TLS proxy"
+maintainer="Jannis Christ <hello@jannis.ovh>"
+license="2-clause-BSD"
+homepage="https://hitch-tls.org"
+distfiles="https://hitch-tls.org/source/hitch-${version}.tar.gz"
+checksum=d6118a3bf0325fc7d7635eec2c467364ac4bac3c7584b2f194067714ba36453d
+system_accounts="hitch"
```
Comment (Vaelatern) [inline srcpkgs/hitch/template]: We seem to be trending to using `_` to denote system level fields, so `system_accounts="_hitch"` and other relevant changes.

Comment (jannisch) [inline srcpkgs/hitch/template]: So in this case just the `system_accounts` field, right? I'm not completely sure what you mean.

Comment (Vaelatern) [inline srcpkgs/hitch/template]: Looks like you made the right changes to me.

**Pull request closed without merging**